semantics/psc-trees-while_stmt.ads

```diff
@@ -50,6 +50,9 @@ package PSC.Trees.While_Stmt is
       New_Operand : Optional_Tree);
    --  Set Nth operand of given Tree
 
+   function Kind (T : Tree) return Tree_Kind_Enum;
```

Comment:
For a function as simple as this, it might be easier to make these "expression functions".  For example:
`function Kind (T : Tree) return Tree_Kind_Enum is (While_Stmt_Kind);`